### DISCUSSION TITLE
missing g?

### DISCUSSION THREAD

**Discussion by fredmonroe** [nautechsystems/nautilus_trader#1661] [Q&A] [answered]
is this a bug?
nautilus_trader/core/nautilus_pyo3.pyi

class OmsType(Enum):
    UNSPECIFIED = "UNSPECIFIED"
    NETTING = "NETTING"
    HEDGING = "HEDGIN"   <---- G missing?

**Comment by cjdsellers** [comment, marked as answer]
Hi @fredmonroe

Definitely a bug, thanks for reporting.

The reason it hasn't caused any issues yet is because these are the type stubs for the pyo3 based Python bindings, which aren't properly in use yet.

I've fixed this on `develop` branch.
